roles/openshift_master_facts/tasks/main.yml

```diff
@@ -32,7 +32,7 @@
   openshift_facts:
     role: master
     local_facts:
-      cluster_method: "{{ openshift_master_cluster_method | default(None) }}"
+      cluster_method: "{{ openshift_master_cluster_method | default('native') }}"
```

Comment:
Thanks for that, no more `openshift_master_cluster_method variable not set` after 30 minutes of installation.